bin/rename_contigs.py

```diff
@@ -36,8 +37,11 @@ def rename_contigs(fasta, assembler, output):
         with gzip.open(fasta, "rt") as handle:
             for seq_record in SeqIO.parse(handle, "fasta"):
                 contig_id = contig_id + 1
-                newid = str(contig_id).zfill(5)
-                header = ">" + name + "_" + assembler + "_contig_" + str(newid) + "\n"
+                newid = str(contig_id).zfill(7)
+                if assembler is not None:
+                    header = ">" + assembler + "_" + name + "_contig_" + str(newid) + "\n"
+                else:
+                    header = ">" + name + "_contig_" + str(newid) + "\n"
```

Comment:
why would assembler be none here ever?

Reply:
if the assembler is already in the filename, since that would get carried through to the contigs in this setup, such as the case of metaspades output